### PR TITLE
fix new generating pipeline file outside the folder

### PR DESCRIPTION
**Pull request by aaksenczuk**
fix problem with new command where pipeline files are created outside the project directory

### CODE REVIEW

File: scargo/file_generators/cicd_gen.py [hunk 1]
```diff
@@ -46,13 +46,12 @@ def merge_dictionaries(
                 dict1[key] = dict2[key]
         return dict1
 
-    def _get_cicd_content(self, path: str, file_name: str) -> Dict[str, Any]:
-        ci_folder = Path(path)
-        file_path = ci_folder / file_name
+    def _get_cicd_content(self, path: Path, file_name: str) -> Dict[str, Any]:
```
Comment (fkuatspyro) [inline scargo/file_generators/cicd_gen.py:49]: Can we give a better name for first argument instead of path? cicd_dir_path or something

**Review by fkuatspyro — DISMISSED** [review]

File: scargo/file_generators/cicd_gen.py (hunk 1 above)
Comment (aaksenczuk) [reply scargo/file_generators/cicd_gen.py]: fixed

File: scargo/file_generators/cicd_gen.py [hunk 2]
```diff
@@ -46,13 +46,12 @@ def merge_dictionaries(
...
+        file_path = path / file_name
 
         # Check if the ci folder exists. If not, create it.
-        if not ci_folder.exists():
-            ci_folder.mkdir(parents=True)
+        if not path.exists():
```
[the remark this answers is not in the document]
Comment (aaksenczuk) [reply scargo/file_generators/cicd_gen.py]: fixed

File: scargo/file_generators/cicd_gen.py [hunk 3]
```diff
@@ -46,16 +46,15 @@ def merge_dictionaries(
...
 
         # Check if the ci folder exists. If not, create it.
-        if not ci_folder.exists():
-            ci_folder.mkdir(parents=True)
+        if not cicd_dir_path.exists():
+            cicd_dir_path.mkdir(parents=True)
 
         # Check if file exists. If not, create it.
-        if not file_path.exists():
+        if not file_path.is_dir():
```
Comment (fkuatspyro) [inline scargo/file_generators/cicd_gen.py:57]:
```suggestion
        if not file_path.is_file():
```

File: scargo/file_generators/cicd_gen.py [hunk 4]
```diff
@@ -46,16 +46,15 @@ def merge_dictionaries(
...
-    def _get_cicd_content(self, path: str, file_name: str) -> Dict[str, Any]:
-        ci_folder = Path(path)
-        file_path = ci_folder / file_name
+    def _get_cicd_content(self, cicd_dir_path: Path, file_name: str) -> Dict[str, Any]:
+        file_path = cicd_dir_path / file_name
 
         # Check if the ci folder exists. If not, create it.
-        if not ci_folder.exists():
-            ci_folder.mkdir(parents=True)
+        if not cicd_dir_path.exists():
```
Comment (fkuatspyro) [inline scargo/file_generators/cicd_gen.py:53]:
```suggestion
        if not cicd_dir_path.is_dir():
```

File: scargo/file_generators/cicd_gen.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (fkuatspyro) [reply scargo/file_generators/cicd_gen.py]: You changed in wrong place _file_path_ instad of dir. I added new comments for it

File: scargo/file_generators/cicd_gen.py (hunk 4 above)
Comment (aaksenczuk) [reply scargo/file_generators/cicd_gen.py]: fixed

File: scargo/file_generators/cicd_gen.py (hunk 3 above)
Comment (aaksenczuk) [reply scargo/file_generators/cicd_gen.py]: fixed

**Review by fkuatspyro — APPROVED** [review]

**Pull request merged**